docs/source/commands/contracts.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: Rover contract commands
+description: For use with Apollo Contract variants
+---
+
+A **contract** is a variant that is the result of filtering on a source variant:
+
+Rover commands that interact with contracts begin with `rover contract`.
+
+## Describing a contract variant
+
+These commands enable you to describe a contract in a federated graph.
+
+### `contract describe`
+
+TODO(caydie) is this correct? I'm assuming you always have to authn to rover with studio 
```

Comment:
Yes, this is correct. You always have to authenticate Rover with Studio when a graph ref is provided. Commands with no graph ref do not require auth (`rover dev` and `rover supergraph compose` are the main ones but there are some others).

Reply:
thank you!!